docs/install/uninstallEverest.md

```diff
@@ -1,53 +1,59 @@
 # Uninstall Everest
 
-No matter why you need to uninstall Everest, you can run the commands below to remove all the Everest resources including: 
+No matter why you need to uninstall Everest, you can run the commands below to remove all the Everest resources including:
 
 - All Kubernetes objects created by Everest
 - All docker containers created by Everest
 - All downloaded binaries and files like **everestctl**, **quickstart.yml**.
 
-
+!!! note alert alert-primary "Warning"
+  Performing these actions will remove all database clusters and data from the Kubernetes cluster
 To uninstall Everest:
 {.power-number}
 
-1. Remove ALL PVCs: 
+1. Remove ALL PVCs:
 
     ```sh
     kubectl delete pvc --all -n percona-everest
     ```
 
 2. Remove ALL created database clusters:
-    
+
     ```sh
     kubectl delete db --all -n percona-everest
     ```
 
 3. List CSVs and remove those with a "*percona*" and "*everest*" prefix, or remove ALL operators:
 
-    ```sh 
+    ```sh
     kubectl delete sub --all -n percona-everest
     kubectl delete ip --all -n percona-everest
     kubectl delete csv --all -n percona-everest
     ```
 
-4. Remove ALL created CRD individually, or use: 
+4. Remove ALL created CRD individually, or use:
 
     ```sh
     kubectl get crd -n percona-everest -o name | grep .percona.com$ | awk -F '/' {'print $2'} | xargs --no-run-if-empty kubectl delete crd -n percona-everest
```

Comment:
As the CRD is cluster scoped, we can remove the `-n percona-everest`

Reply:
We do this already

Reply:
I mean `kubectl get crd -n percona-everest` is same as `kubectl get crd`